### DISCUSSION TITLE
I have a problem with the video player sliding to advance the video is having a problem

### DISCUSSION THREAD

**Discussion by DENIS-CAMICAZE** [ReVanced/revanced-discussions#909] [General]
https://user-images.githubusercontent.com/125999493/221578974-71526508-26b4-48b9-a472-60d36f302926.mp4

1. as I show in the video, it does not help three seconds seconds only from 30 seconds to one minute onwards and in previous versions it worked normally it could be a second advancing the video with the finger sliding

**Comment by oSumAtrIX** [comment]
It snaps, simply hold and then let go

**Reply by DENIS-CAMICAZE** [reply]
Yes oSum , I hold the drag and drop but if it's something of 3 seconds the video doesn't jump just above 30 seconds you have to drag a lot for it to advance the video

**Reply by oSumAtrIX** [reply]
I don't understand this sentence

**Comment by DENIS-CAMICAZE** [comment]
https://user-images.githubusercontent.com/125999493/221607826-aaf77125-7396-4787-96f5-ad373534e57c.mp4

the video only works if you skip 30 seconds but if you skip 3 seconds or 5 seconds the video doesn't go

**Reply by oSumAtrIX** [reply]
You have to long press. This is default behavior of YouTube

**Comment by DENIS-CAMICAZE** [comment]
I can not advance the video 3 seconds? now I can only advance the video 1 minute forward

**Reply by oSumAtrIX** [reply]
I don't understand this sentence.

**Reply by DENIS-CAMICAZE** [reply]
estou escrevendo em português Traduz para Inglês por favor , eu quero dizer que agora eu não posso mais avançar o vídeo 3 segundos , eu só posso avançar o vídeo de 1 minuto  para frente

**Reply by DENIS-CAMICAZE** [reply]
can you translate to english ?

**Reply by oSumAtrIX** [reply]
Please translate beforehand next time. As previously said, you have to hold a second and then let go to disable snapping.

**Reply by DENIS-CAMICAZE** [reply]
oSum did several tests and realized that this problem only happens on Android 12 and Android 13 of being able to advance the video by sliding with your finger, it cannot advance 2 seconds, only 30 seconds forward or 1 minute forward, this problem is only on Android 12 and Android 13 , on Android 10 and Android 11 you touch the part of the video you want with your finger and it works , This of video is on android 10 , Patch 164 

https://user-images.githubusercontent.com/125999493/221725581-3f5945e6-ba67-4ef5-8b85-08db2bcaec4e.mp4

**Reply by oSumAtrIX** [reply]
Exclude patches, report back which one makes it behave like this.

**Reply by DENIS-CAMICAZE** [reply]
Ok

**Comment by DENIS-CAMICAZE** [comment]
oSum make a video please demonstrating how to advance at least 2  seconds with your finger sliding

**Comment by DENIS-CAMICAZE** [comment]
oSum now the only way to advance 2 seconds of the video now, it's just the way I'm showing it in this video

https://user-images.githubusercontent.com/125999493/221631406-987f640d-100d-4006-8e46-653ee73ae568.mp4
